vscode-lean4/src/infoview.ts

```diff
@@ -203,6 +269,11 @@ export class InfoProvider implements Disposable {
         this.serverNotifSubscriptions.clear();
     }
 
+    private clearSessions() {
```

Comment:
`clearRpcSessions`? More to the point, `infoview.ts` is becoming a bit of a catch-all blob. What do you think about putting the RpcSession management in a utility class somewhere that `infoview` calls?

Reply:
Indeed the InfoProvider class is getting more and more complicated.  But splitting up the file won't help with that.

Reply:
Perhaps we will need to refactor it then at some future point, but it is low priority.